Source/Core/Core/PowerPC/PowerPC.h

```diff
@@ -133,6 +134,7 @@ extern PowerPCState ppcState;
 extern Watches watches;
 extern BreakPoints breakpoints;
 extern MemChecks memchecks;
+extern CodePatches code_patches;
```

Comment:
Not really a fan of adding Yet Another Global™ to the existing problem of global state.